analyses/0007-porter.md

```diff
@@ -1,18 +1,8 @@
 ---
-title: Porter tech docs assessment draft
-tags: porter
+title: Porter docs analysis
+cSpell:ignore: Uchechukwu Obasi
 ---
 
-# Notes
-
-meeting notes:
-https://docs.google.com/document/d/12OGtSaUtlc7OA_iPnUvmVaiKg8yM7_QBzFnMgoHvnLw/edit?usp=sharing
-website: https://getporter.org/ repos:
-
-- https://github.com/getporter/porter (main site), site content is in docs/
-  folder
-- https://github.com/getporter/operator (https://getporter.org/operator site)
-
```

Comment:
Move the "Notes" into "Resources" further below, since we want to avoid multiple h1 headings.